content/docs/architecture_guide/security.md

```diff
@@ -212,10 +220,10 @@ Some platforms leverage the same virtualization hardware as VMs in order to
 provide better system call interception performance. However, gVisor does not
 implement any device emulation, and instead opts to use a sandboxed host System
 API directly. Both approaches significantly reduce the original attack surface.
-Ultimately, since gVisor uses the same hardware mechanism, one should not assume
-that the mere use of virtualization hardware makes a system more or less secure,
-just as it would be a mistake to make the claim that the use of an engine makes
-a car safe.
+Ultimately, since gVisor is capable of using the same hardware mechanism, one
```

Comment:
I still find this sentence confusing though I don't have a ready alternative. I'm not quite sure of the point. Is it that no one singular thing makes the system secure (in this case virtualization hardware) but multiple things?

It sounds like we're saying "gVisor *could* use virtualization hardware + device emulation but doesn't and that makes gVisor's security on par with VMs". I don't get how "gVisor is capable of using the same hardware mechanism" has any bearing on anything in this sentence.

Reply:
Yeah, the idea is that the existence of VMX hardware itself doesn't make a system secure. Just as we wouldn't assert that KVM mode makes gVisor security.

I will work to clarify this again later and push these changes for now, since I think it's an improvement.